quattor/types/hardware.pan

```diff
@@ -175,6 +175,21 @@ type structure_sysloc = {
 };
 
 @documentation{
+    System benchmark results
+    benchmarks is used to hold the performance benchmark for the machine 
+    i.e. HEPSpec06 score
+    this might be used to scale things such as the wall time
```

Comment:
i'm not sure the example adds much (the variable is very specific, and this is core templates after all). but i can't come up with a better example, so feel free to leave it.